File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/LetterAttachmentDB/LetterAttachment/LetterAttachmentForm.cs
using Serenity.ComponentModel;
using System;

namespace CorrespondenceSystem.LetterAttachmentDB.Forms;

[FormScript("LetterAttachmentDB.LetterAttachment")]
[BasedOnRow(typeof(LetterAttachmentRow), CheckNames = true)]
public class LetterAttachmentForm
{
    public string Title { get; set; }
    public string AttachmentFile { get; set; }

}

File: CorrespondenceSystem/CorrespondenceSystem.Web/Modules/ServerTypes/LetterAttachmentDB/LetterAttachmentForm.ts
import { StringEditor, ImageUploadEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface LetterAttachmentForm {
    Title: StringEditor;
    AttachmentFile: ImageUploadEditor;
}

export class LetterAttachmentForm extends PrefixedContext {
    static readonly formKey = 'LetterAttachmentDB.LetterAttachment';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!LetterAttachmentForm.init)  {
            LetterAttachmentForm.init = true;

            var w0 = StringEditor;
            var w1 = ImageUploadEditor;

            initFormType(LetterAttachmentForm, [
                'Title', w0,
                'AttachmentFile', w1
            ]);
        }
    }
}